conf/nebula-metad.conf.default

```diff
@@ -13,6 +13,11 @@
 --v=0
 # Maximum seconds to buffer the log messages
 --logbufsecs=0
+# Whether to redirect stdout and stderr to separate output files
+--redirect_stdout=true
+# Destination filename of stdout and stderr, which will also reside in log_dir.
+--stdout_log_file=metad-stdout.log
```

Comment:
logs/metad-stdout.log?

Reply:
`setupLogging` will join the `FLAGS_log_dir` and `FLAGS_stdout_log_file` paths